packages/editor/src/lib/exports/StyleEmbedder.ts

```diff
@@ -27,7 +32,7 @@ export class StyleEmbedder {
 	readRootElementStyles(rootElement: Element) {
 		// when reading a root, we always apply _all_ the styles, even if they match the defaults
 		this.readElementStyles(rootElement, {
-			shouldRespectDefaults: false,
+			shouldRespectDefaults: true,
```

Comment:
this is the controversial line i'd like to understand/discuss

Reply:
From the comment above this, I _think_ it's because this is a root element? Which means that in order for the inheritance stuff to be set up we need to pro-actively apply styles to it?

I think I would test this a bit with global/inherited styles. I believe there were some quirks I found with this when you're also using tailwind?

Reply:
could also be worth tweaking this so that it only applies defaults that may later be inherited from?